index.js

```diff
@@ -1 +1,46 @@
-// Your code here
+$(document).ready(() => {
+
+  const digitalClock = () => {
+    let currentTime = new Date();
+    let month = currentTime.getMonth();
+    let date = currentTime.getDate();
+    let year = currentTime.getFullYear();
+    let weekday = new Array(7);
+    weekday[0] = "Sunday";
+    weekday[1] = "Monday";
+    weekday[2] = "Tuesday";
+    weekday[3] = "Wednesday";
+    weekday[4] = "Thursday";
+    weekday[5] = "Friday";
+    weekday[6] = "Saturday";
```

Comment:
This could probably be simplified by doing something like 
const weekday = ['Sunday', 'Monday'...'Saturday']